app/enterprise/1.3-x/deployment/installation/amazon-linux.md

```diff
@@ -1,321 +1,312 @@
-  ```
+    ```bash
+    $ sudo cp /etc/kong/kong.conf.default /etc/kong/kong.conf
+    ```
 
-  Export the KONG_PASSWORD environment variable:
-  ```
-  export KONG_PASSWORD=kong
-  ```
+2. Uncomment and update the PostgreSQL database properties in `/etc/kong/kong.conf` using your preferred text editor. Replace pg_user, pg_password and pg_database with the values:
 
-### Enable Kong Manager
+    ```
+    pg_user = kong
+    pg_password = kong
+    pg_database = kong
+    ```
 
-  Kong Enterprise's Graphical User Interface **Kong Manager** can be connected by setting the `admin_gui_url` property to the EC2 instances `IPv4` address and the `admin_listen` property to allow **Kong Manager** to communicate with the Admin API.
+    > Note: If you used different values for the user and database name, use those values for the user and database name. 
 
-  ```
-  admin_gui_url = http://ec2-xx-xxx-xx-xx.us-east-2.compute.amazonaws.com:8002
-  admin_listen = 0.0.0.0:8001, 0.0.0.0:8444 ssl
-  ```
+## Step 5. Seed the Super Admin password and boostrap Kong
 
-### Enable the Dev Portal
+Setting a password for the **Super Admin** before initial start-up is strongly recommended. This will permit the use of RBAC(Role Based Access Control) at a later time, if needed.
 
-  Kong Enterprise's **Developer Portal** can be connected by setting the `portal` property to `on` and setting the `portal_gui_host` property to the EC2 instance's `IPv4` address
+1. Create an environment variable with the desired **Super Admin** password and keep password in a safe place:
 
-  ```
-  portal = on
-  portal_gui_host = http://ec2-xx-xxx-xx-xx.us-east-2.compute.amazonaws.com:8003
-  ```
+    ```bash
+    $ export KONG_PASSWORD=<password-only-you-know>
+    ```
 
+2. Run migrations to prepare the Kong database.
 
-## Step 5. Start Kong
+    ```bash
+    $ sudo /usr/local/bin/kong migrations bootstrap -c /etc/kong/kong.conf
+    ```
 
-1. Run migrations
-  ```
-  kong migrations bootstrap -c /etc/kong/kong.conf
-  ```
-2. Start Kong
-  ```
-  sudo /usr/local/bin/kong start -c /etc/kong/kong.conf
-  ```
+3. Start Kong Enterprise:
 
+    ```bash
+    $ sudo /usr/local/bin/kong start -c /etc/kong/kong.conf
+    ```
 
-## Step 6. Verify Kong is Receiving Requests
+4. Verify Kong Enterprise is working:
 
-##### Install HTTPie to easily send requests to Kong *(optional)*
-  ```
-  sudo yum install python36
+    ```bash
+    $ curl -i -X GET --url http://localhost:8001/services
+    ```
+    
+5. You should receive a `HTTP/1.1 200 OK` message.
 
-  sudo pip-3.6 install --upgrade pip setuptools
+## Step 6. Finalize Configuration and Verify Installation
 
-  sudo /usr/local/bin/pip3.6 install --upgrade httpie
-  ```
+### Enable and Configure Kong Manager
 
-#### Test your connection to Kong
+1. To access Kong Enterprise's Graphical User Interface, Kong Manager, update the `admin_gui_url` property in `/etc/kong/kong.conf` file the to the DNS, or IP address, of the CentOS system. For example:
```

Comment:
This part of the sentence is missing something "file the to the DNS, or IP address, of the CentOS system. For example:"  

Also, is CentOS correct, as this is an amazon-linux install?